### ISSUE TITLE
Missing example.

### ISSUE THREAD

**Issue by codeHatcher** [mbell8903/passport-custom#4]
Your example has gone missing. I would re-add just to help people out a little bit to get started. For instance I wasn't sure if I had to initialize it 

via (right)

```javascript
var CustomStrategy = require('passport-custom').Strategy;
```
or via (wrong)

```javascript
var CustomStrategy = require('passport-custom').CustomStrategy;
```
It took me two seconds to look at the library and figure it out but others might not spend anytime doing that and discard the library.

Thanks for making this, it definitely saved me some time.

**Comment by angelxmoreno** [comment]
@codeHatcher after being annoyed at other strategies for not including that part, I realized that they all follow the same syntax. I wish the docs mentioned it.

[passport-http-bearer](https://github.com/jaredhanson/passport-http-bearer/blob/master/lib/index.js#L15) and pretty much all done by @jaredhanson

[passport-saml](passport-http-bearer)

I still say we should have this info for noobs like myself!

**Comment by mbell8903** [comment]
Fixed in ef33364b1336d45d5a9c95eb4b2f818c64a8e597
